### PR TITLE
Add reset script for raspberry pi

### LINKED ISSUE #170: Add script to automatically reset the photobooth
**Is your feature request related to a problem? Please describe.**
After using the photobooth on an event, it is very annoying to manually reset it (remove pictures, remove generated qr codes, reset database, ...).

**Describe the solution you'd like**
A simple shell script that will reset your photobooth for you.

### PR DESCRIPTION
**Pull request by mhellmeier**
Fix #170 

This script will ask the user if he or she wants to

- reset the system to factory settings (this will remove the `data` folder and the `my.config.inc.php` file)
or
- only reset the data (this will only remove the `data` folder)

The code style is oriented on the installation script (https://github.com/andreknieriem/photobooth/blob/master/install-raspbian.sh)

### CODE REVIEW

**Comment by mhellmeier** [comment]
If this PR will be accepted it would be great if someone can mention this sh script in the wiki.

It can be executed with `sudo bash reset-raspbian.sh` on Raspbian.

File: reset-raspbian.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    info "### Ok, lets go and remove some things."
+
+    info "### Remove your data ..."
+    rm -rf data
+    mkdir data
+    chown -R www-data:www-data data
+
+    info "### Remove your config file ..."
+    rm -rf config/my.config.inc.php
+    chown -R www-data:www-data data
```
Comment (andi34) [inline reset-raspbian.sh]: Chowned in line 43 already

Comment (mhellmeier) [reply reset-raspbian.sh]: Whooops, corrected! 😉

**Comment by sualko** [comment]
It's similar to the reset function in the admin settings, but some people (like me) still like a shell option.

**Pull request merged**